setup.py

```diff
@@ -46,6 +46,7 @@ def run(self):
     'executor',
     'jinja2',
     'babel',
+    'weasyprint'
```

Comment:
How about making this optional ?

```python
extras_require={
        'weasyprint': ['weasyprint'],
}
```

and then we could use ``pip install openlabs_report_webkit[weasyprint]``